parts/linux/cloud-init/artifacts/configure_azure0.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Check if the azure cni config is there... no need to run this script if not
+# Also don't want to run this when not using azure-cni
+[ ! -f /etc/cni/net.d/10-azure.conflist ] && exit 0
+
+# Check if the azure0 bridge is already configured
+# We don't need to run if so.
+ip link show azure0 && exit 0
+
```

Comment:
CNI team mentions that this is not needed for calico network policy, so we should check the network policy and skip this configuration. The network policy is a parameter in cse_cmd.sh, we can get the value in the similar way

Reply:
Tried to use "{{GetParameter "networkPolicy"}}" in nodecustomdata.yml. Is it the right way to pass the network policy as a parameter?